modules/java_api/tests/TensorTests.java

```diff
@@ -5,11 +5,22 @@
 import org.intel.openvino.*;
 import org.junit.Test;
 
+import java.util.Random;
+
 public class TensorTests extends OVTest {
+    int[] dimsArr = {1, 3, 128, 128};
+
     @Test
     public void testGetTensorFromFloat() {
-        int[] dimsArr = {1, 1, 2, 2};
-        float[] data = {0.0f, 1.1f, 2.2f, 3.3f};
+        int size = 1;
+        for (int val : dimsArr) size *= val;
+
+        float[] data = new float[size];
+
+        Random rand = new Random();
+        for (int i = 0; i < size; i++) data[i] = rand.nextFloat();
```

Comment:
It's better to keep using fixed data, not random.

Reply:
Randomizer take more resources. Also, data volume os small and we can keep it explicitly initialized.